### ISSUE TITLE
Request: NestJS wrapper

### ISSUE THREAD

**Issue by arneschreuder** [sammysaglam/federation-with-subscriptions#2]
Thank you for the good work. Can we please get a NestJS wrapper for this? I actually came across this entire issue through Nest's implementation of apollo-graphql. I would be happy to help?

**Comment by sammysaglam** [comment]
yeah, agreed, definitely would be good to have support for more frameworks. ill try to take a look sometime soon, but in the meantime please feel free to submit a PR or even a partial solution to build upon.
